pkg/volume/rbd/rbd_util.go

```diff
@@ -269,7 +270,11 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) error {
 
 		// fence off other mappers
 		if err = util.fencing(b); err != nil {
-			return fmt.Errorf("rbd: image %s is locked by other nodes", b.Image)
+			if err.Error() == rbdCmdErr {
+				return fmt.Errorf("rbd: rbd cmd not found")
+			} else {
+				return fmt.Errorf("rbd: image %s is locked by other nodes", b.Image)
+			}
```

Comment:
The messages in the 'else' are different, so if we want to wrap it we need to make the function like **rbdErrors(err error, format string, args ...interface{})**. It seems complicated...
And sometimes we don't want to return it, such as line 447.
Thanks.